src/portable/ReactUi/react/src/components/NavPanel.tsx

```diff
@@ -20,12 +20,13 @@ class NavPanel extends React.Component<IProps, object> {
         const user = users.filter(u => u.username.id === selectedUser)[0];
         const admin = user && user.role && user.role.filter(r => r === "administrator")[0];
         const project = tasks.filter(t => t.id === selectedProject)[0];
+        const backLinkWrapper = admin !== null && tasks.length === 1 && users.length === 1? "":<BackLink target="/" />
         const userAvatar = user? (
             <AvatarLink id={user.username.id}
                 name={user.displayName}
                 target="/settings"
                 uri={user.username.avatarUri? user.username.avatarUri: ""} />):"";
-        const projectClick = admin != null? "/ProjectSettings": "/main"
+        const projectClick = admin !== null? "/ProjectSettings": "/main"
```

Comment:
I think this logic was put in to handle the case where admin is undefined. This will throw an error if admin is undefined although with the change you made above, it probably will throw an error before this line so in fact this change wouldn't matter but I think the code is better before the change...